production/helm/loki/CHANGELOG.md

```diff
@@ -0,0 +1,47 @@
+# Changelog
+
+All notable changes to this library will be documented in this file.
+
+Entries should be ordered as follows:
+
+- [CHANGE]
+- [FEATURE]
+- [ENHANCEMENT]
+- [BUGFIX]
+
+Entries should include a reference to the pull request that introduced the change.
+
```

Comment:
since the chart is in a different location now, I'm not sure if the versioning should be kept. Anyway, just a thought: maybe we should try using a versioning like `x-y`, were x is the related loki version and y is the chart version. So this chart would be `2.6.0-1` or something. I feel like semver doesn't suit it very well idk

Reply:
agreed, nuked it to just have an entry for `3.0.0` mentioning the deprecation with links back to the original locations